scripts/create_py_tags.py

```diff
@@ -293,7 +303,10 @@ def main():
     parser.add_builtins()
 
     for filename in args:
-        parser.process_file(filename)
+        try:
+            parser.process_file(filename)
+        except (SystemExit, ImportError, TypeError):
```

Comment:
Why catching `SystemExit` and `ImportError`? They do not occur, at least not for me.
The `TypeError` exceptions seem rather like an error we should handle instead of silently ignoring.

In either case, I think we should print the error instead of silencing it:
```python
        except TypeError as exc:
            print(f'{exc.__class__.__name__} in {filename}: {exc}')
```
this gives: `TypeError in /home/enrico/.pyenv/versions/3.9.0/lib/python3.9/http/cookies.py: can only concatenate list (not "tuple") to list`